manifest.json

```diff
@@ -2,7 +2,7 @@
     "spark_version": "2.2.1",
     "default_spark_dist": {
         "hadoop_version": "2.6",
-        "uri": "https://downloads.mesosphere.com/spark/assets/spark-2.2.1-2-bin-2.6.tgz"
+        "uri": "svt-dev.s3.amazonaws.com/spark/spark-2.2.1-bin-2.6.5-DCOS-38138.tgz"
```

Comment:
I guess this will be updated once the next release is cut?

Reply:
Yup, or do you think I should first merge the [mesosphere/spark PR](https://github.com/mesosphere/spark/pull/33) (and revert the --proxy-user commit) off `custom-branch-2.2.1-X`? I could then create a new tgz and upload it to 
`downloads.mesosphere.com/spark/assets/spark-2.2.1-3-bin-2.6.tgz` (note the `-3`) and change this PR. That way I wouldn't have to do it twice?

Reply:
Hm, I don't think we should keep overwriting the "2.2.1-3" tgz with different versions. Until we actually make a release, how about something like "2.2.1-3-SNAPSHOT" to signal that it's not an officially released version, but a dev one that will keep changing up until the release.

Reply:
SNAPSHOT sounds best